git-poem.txt

```diff
@@ -1,5 +1,3 @@
-there once was a man from minnetonka
-This would be a lot easier for me if it were code and not prose
-I can't think of anything that rhymes with Minnetonka
-It's going to be a lame ass poem
-How about "Willy Wonka"?
+There once was a man from Minnetonka
+His chocolates were better than Wonka
```

Comment:
I wanted "Wonka" to be "Wonka's" but I'm not sure the syllables are correct for a limerick.  What do you think @davidtcalabrese

Reply:
I think "Wonka's" works great. Limericks are frequently non-sensical so I wouldn't be concerned too much with breaking the rules. I think we should maintain the traditional _aabba_ rhyme scheme and try our best to match the anapestic trimeter. 

Unlike in programming, poems aren't broken when the syntax is changed slightly, and sometimes they are enhanced. Shakespeare was famous for writing in iambic pentameter but if you look closely at his plays you see he strays from it all the time! Some lines have added syllables, some are less than ten syllables. Sometimes he just wrote in prose! I had a theater professor who said that straying from this rigidity in drama makes it seem more natural.

Reply:
Excellent. I will update the PR to reflect this advice.

Reply:
So is this ready to be resolved and merged? What is the typical process for doing this?

Reply:
I have not made the proposed change yet. (it's still Wonka and not Wonka's) Once that change is made, then you can approve the PR and then merge it.